### ISSUE TITLE
resnet,  unexpected keyword argument 'freeze_bn'

### ISSUE THREAD

**Issue by sevenseablue** [delftrobotics/keras-retinanet#16]
pip install keras_resnet

keras_retinanet/models/resnet.py
resnet = keras_resnet.models.ResNet50(image, include_top=False, freeze_bn=True)
error
__init__() got an unexpected keyword argument 'freeze_bn'

**Comment by hgaiser** [comment]
Hi there,

`keras-retinanet` depends on a modified version of `keras-resnet`, which is not yet merged. If you install [this](https://github.com/delftrobotics-forks/keras-resnet/tree/bn-freeze) branch instead it should work.

**Comment by kaishijeng** [comment]
I install https://github.com/delftrobotics-forks/keras-resnet/tree/bn-freeze and still get the same error
Can you verify?

Thanks,

**Comment by kaishijeng** [comment]
Ignore this error because I installed a wrong branch

**Comment by sakares** [comment]
@kaishijeng Did you install from branch "bn-freeze"? I still got an error. Thanks

**Comment by hgaiser** [comment]
Since this issue, the PR for BatchNorm freezing has been merged into https://github.com/broadinstitute/keras-resnet . So you should clone that master.
